teamB/cheonsongin/app.js

```diff
@@ -1,34 +1,17 @@
+require('dotenv').dotenv.config();
```

Comment:
해당 코드로 서버를 실행했을 시 오류가 발생할 것입니다.
수정해주세요!
```suggestion
require('dotenv').config();
```